runtime/common/src/lib.rs

```diff
@@ -47,18 +47,26 @@ pub mod weights;
 macro_rules! decl_common_types {
     {} => {
         use sp_runtime::generic;
-        use frame_support::traits::{Currency, Imbalance, OnRuntimeUpgrade, OnUnbalanced, NeverEnsureOrigin, TryStateSelect};
+        #[cfg(feature = "try-runtime")]
+        use frame_try_runtime::{UpgradeCheckSelect, TryStateSelect};
+        use frame_support::traits::{Currency, Imbalance, OnRuntimeUpgrade, OnUnbalanced, NeverEnsureOrigin};
 
         pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 
         type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
+        type Migrations = (
+            pallet_contracts::Migration<Runtime>,
+            pallet_scheduler::migration::v4::CleanupAgendas<Runtime>,
+        );
```

Comment:
How did you find out which pallets need a migration?

Reply:
Lastly, I verified whether the migrations are relevant to us. Most of the time they are mandatory, sometimes there also migrations for specific contexts and goals.